### PR TITLE
Update Clear Command to clear current list

### PR DESCRIPTION
**Pull request by Weiennn**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ClearCommand.java [hunk 1]
```diff
@@ -2,22 +2,29 @@
...
 
 
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
-        model.setAddressBook(new AddressBook());
+        List<Person> shownList = model.getFilteredPersonList();
+
+        while (shownList.size() != 0) {
+            Person personToDelete = shownList.get(0);
```
Comment (aexolate) [inline src/main/java/seedu/address/logic/commands/ClearCommand.java:25]: The code works fine and is understandable, but would the '0' might be considered magic number? Perhaps a stream based approach could improve readability and changing it to `while(!shownList.isEmpty())` can improve clarity.

**Review by aexolate — APPROVED** [review]
Did a quick test and the command seems to work fine. It would probably be a good idea to mention the other changes (phone number) in the description of the PR since the title of the PR suggests it is only related to the clear command.

**Comment by Weiennn** [comment]
Also changed the numbers in the test cases. Some numbers were 7 digits long while some numbers were 8, changed it so that all of it were 8 digits long to be more consistent

**Comment by aexolate** [comment]
LGTM.

**Pull request merged**